services/harmony/app/frontends/health.ts

```diff
@@ -1,8 +1,11 @@
 import { Response, NextFunction } from 'express';
+import { isCmrHealthy } from '../util/cmr';
+import { isEdlHealthy } from '../util/edl-api';
 import HarmonyRequest from '../models/harmony-request';
 import { Job } from '../models/job';
 import logger from '../util/log';
-import db, { Transaction } from '../util/db';
+import RequestContext from '../models/request-context';
+import db from '../util/db';
 
 const UP = 'up';
```

Comment:
an enumerated type would be better here rather than strings. That way you get type checking that can maker sure someone doesn't accidentally return an arbitrary string instead of using one of these

Reply:
Changed to enum.